quickwit/quickwit-metastore/src/metastore/file_backed/file_backed_index/shards.rs

```diff
@@ -245,9 +245,10 @@ impl Shards {
     ) -> MetastoreResult<MutationOccurred<PruneShardsResponse>> {
         let initial_shard_count = self.shards.len();
 
-        if let Some(max_age) = request.max_age {
+        if let Some(max_age_secs) = request.max_age_secs {
             self.shards.retain(|_, shard| {
-                let limit_timestamp = OffsetDateTime::now_utc().unix_timestamp() - max_age as i64;
+                let limit_timestamp =
+                    OffsetDateTime::now_utc().unix_timestamp() - max_age_secs as i64;
                 shard.update_timestamp >= limit_timestamp
```

Comment:
nitpick
```suggestion
                let now = OffsetDateTime::now_utc().unix_timestamp();
                shard.update_timestamp + max_secs as i64 >= now
```

or if you want the extra name
```suggestion
                let gc_deadline = shard.update_timestamp + max_secs as i64;
                let now = OffsetDateTime::now_utc().unix_timestamp();
                gc_deadline >= now
```